dppy/exotic_dpps.py

```diff
@@ -466,6 +464,12 @@ class UST:
     """
 
     def __init__(self, graph):
+        # For Uniform Spanning Trees
+        try:
+            import networkx as nx
+            self.nx = nx
```

Comment:
Hi @danielecc,
Why the additional `self.nx` ?

Reply:
see above